screens/Menu/ShuttleBus.js

```diff
@@ -1,32 +1,245 @@
+                    hour: element.hour,
+                    minutes: element.minutes
+                });
+            });
+        }
+        return results;
+    };
+
+    /**
+     * Function that sorts through the shuttle bus schedule and compares the time/day to the current time/day
+     */
+    const getNextStops = () => {
+        var scheduleTimesSGW = [];
+        var scheduleTimesLoyola = [];
+        const sgwCampus = "SGW";
+        const loyolaCampus = "Loyola";
+        if (currentDay && weekDays.includes(currentDay.toString()) && !friday.includes(currentDay.toString()) && !weekends.includes(currentDay.toString())) {
+            scheduleTimesSGW = getShuttleBusTimes[sgwCampus].MondayToThursday;
+            scheduleTimesLoyola = getShuttleBusTimes[loyolaCampus].MondayToThursday;
+        }
+        else if (currentDay && friday.includes(currentDay.toString())) {
+            scheduleTimesSGW = getShuttleBusTimes[sgwCampus].Friday;
+            scheduleTimesLoyola = getShuttleBusTimes[loyolaCampus].Friday;
+        }
+        else {
+            return alert("There is no shuttle bus on weekends. Please check back during the week!");
+        }
+        var nextStopsSGW = fetchNextStops(scheduleTimesSGW);
+        var nextStopsLoyola = fetchNextStops(scheduleTimesLoyola);
+        return ({
+            sgwStops: nextStopsSGW,
+            LoyolaStops: nextStopsLoyola
+        });
+    };
+
+    useEffect(() => {
+        const intervalId = setInterval(() => {
+            getCurrentTime();
+        }, 100);
+        if (currentTime && currentDay && campus) {
+            setResults(getNextStops());
+        }
+        return () => clearInterval(intervalId);
+    }, [currentTime, currentDay, campus]);
+
     return (
         <View style={styles.container}>
-            <View style={styles.menuButtonContainer}>
-                <TouchableOpacity style={styles.menuButton} onPress={goToMenu}>
-                    <Feather name="menu" style={styles.icon} />
-                </TouchableOpacity>
-            </View>
+            <View style={styles.topViewContainer}>
+                <View style={styles.imageContainer}>
+                    <Image style={styles.shuttleImage} source={require("./../../assets/shuttle.jpg")} />
+                </View>
 
-            <Text style={styles.mainLabel}>Shuttle Bus</Text>
+                <View style={styles.menuButtonContainer}>
+                    <TouchableOpacity style={styles.menuButton} onPress={goToMenu}>
+                        <Feather name="menu" style={styles.icon} />
+                    </TouchableOpacity>
+                </View>
+                <View style={styles.titleViewContainer}>
+                    <Text style={styles.mainLabel}>Shuttle Bus</Text>
+                    <Text style={styles.shortLabel}>Winter 2020 Departures</Text>
+                </View>
+            </View>
 
+            <View style={styles.tabContainerView}>
```

Comment:
It would be nice if you have a label that says what is the current day right on top of the Flatlist.